### DISCUSSION TITLE
全给老子去看来自深渊

### DISCUSSION THREAD

**Discussion by ikungei** [share121/inter-knot#1947] [其他]
来自深渊致愈好看
全都去看😡😡😡
![Image_1723557947299](https://github.com/user-attachments/assets/2a388c2d-fde0-4f1b-8296-07a3a5d84dda)

**Comment by AnbulesPeo** [comment]
1111哥们真好看吗，准备找个番

**Reply by ikungei** [reply]
骗你干嘛

**Comment by JohnDoe-A** [comment]
回复楼上1F：好看是真好看，我也非常推荐。一条不能回头的探索之路，一场没有退路的朝圣之旅。但是比较刀，以及漫画作者沉迷原神且身体很不好，所以漫画时常一年两更

**Comment by AOEEVU** [comment]
大好人啊，推荐这么甜的番

**Reply by ikungei** [reply]
我还能骗人？
